### PR TITLE
Add basic scheduler docs

### PR DESCRIPTION
**Pull request by vasilty**

### CODE REVIEW

File: docs/source/scheduler.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- ``args``: args for the task function (optional)
+- ``kwargs``: kwargs for the task function (optional)
+- ``singleton``: Boolean to specify whether this task should not be on the queue
+    multiple times (optional, defaults to ``True``)
+- ``queue``: Queue to put this task on (optional, defaults to
+    ``RT_SCHEDULER_QUEUE``)
+
+``schedule`` for a task can be represented in multiple ways:
+
+1. as a crontab, for example to run a task every Monday you would set::
```
Comment (kaloyanm) [inline docs/source/scheduler.rst:20]: hmm I am confused by the number points (they make me think I need to follow some steps). We could use mini title instead.

File: docs/source/scheduler.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
+Scheduler
+=========
+
+``redis-tasks`` provides developers with an built-in task scheduler.
+A schedule is defined as a setting ``RT_SCHEDULE``, which is a dict of the
+structure ``id`` -> ``entry``. An ``entry`` itself is a dict with the
+following keys:
+
+- ``task``: Import path of the function to be run, e.g. ``mymodule.myfunc``
```
Comment (kaloyanm) [inline docs/source/scheduler.rst:9]: I agree with @i-trofimtschuk  that we can reference the API here somehow. Or I rather would include a full fledged example here.

File: docs/source/scheduler.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+2. as a periodic schedule defined with ``redis_tasks.run_every()``, which
+    accepts the following keyword-only arguments:
+
+    - ``hours``
+    - ``minutes``
+    - ``seconds``
+    - ``start_at``: a string in the format <hours>:<minutes>
+
+    For example if you want your task run every 6 hours starting at 5:00 every
```
Comment (kaloyanm) [inline docs/source/scheduler.rst:32]:
```suggestion
    For instance if you want your task run every 6 hours starting at 5:00 every

```

File: docs/source/scheduler.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- ``task``: Import path of the function to be run, e.g. ``mymodule.myfunc``
+- ``schedule``: Schedule for this task (see below)
+- ``args``: args for the task function (optional)
+- ``kwargs``: kwargs for the task function (optional)
+- ``singleton``: Boolean to specify whether this task should not be on the queue
+    multiple times (optional, defaults to ``True``)
+- ``queue``: Queue to put this task on (optional, defaults to
+    ``RT_SCHEDULER_QUEUE``)
+
+``schedule`` for a task can be represented in multiple ways:
```
Comment (kaloyanm) [inline docs/source/scheduler.rst:18]:
```suggestion
You can ``schedule`` a task in a number of ways:

```

File: docs/source/scheduler.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+1. as a crontab, for example to run a task every Monday you would set::
+
+    "schedule": redis_tasks.crontab("0 0 * * 1")
```
Comment (kaloyanm) [inline docs/source/scheduler.rst:22]: I am still wondering if we should include `RT_SCHEDULER_QUEUE ` inside the examples here.

File: docs/source/scheduler.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
+Scheduler
+=========
+
+``redis-tasks`` provides developers with an built-in task scheduler.
```
[the remark this answers is not in the document]
Comment (vasilty) [reply docs/source/scheduler.rst:4]: This word ships makes the word built-in redundant

File: docs/source/scheduler.rst (hunk 4 above)
Comment (vasilty) [reply docs/source/scheduler.rst:18]: ``schedule`` is a dict key, making it a verb is fun, but confusing

File: docs/source/scheduler.rst (hunk 5 above)
Comment (vasilty) [reply docs/source/scheduler.rst:22]: What exactly do you mean? It is just a string from the scheduler standpoint

File: docs/source/scheduler.rst (hunk 6 above)
Comment (i-trofimtschuk) [inline docs/source/scheduler.rst:4]:
```suggestion
``redis-tasks`` provides developers with a built-in task scheduler.

```

File: docs/source/scheduler.rst (hunk 1 above)
Comment (i-trofimtschuk) [reply docs/source/scheduler.rst:20]: make unnumbered list

**Pull request closed without merging**